docs/usage.md

```diff
@@ -55,6 +55,76 @@ An [example samplesheet](../assets/samplesheet.csv) has been provided with the p
+These files are preferred as they cover the largest amount of the reference genome without including multiple copies of the same sequence which can confuse aligners such as STAR. Most other species (fly, cow, dog etc.) do not have a primary assembly, in which case the complete reference sequence, or "toplevel" assembly, should be used.
+The difference between the two is the inclusion of alternative loci (haplotypes) but these do not typically exist for species outside mouse and human.
+
+### Gene annotation
+
+Gene annotations are updated more frequently than the reference genome sequence and there are more options to consider here.
+Because annotations can be updated frequently, you should rely on sources that include well-defined, versioned releases such as ENSEMBL or GENCODE.
+We generally recommend using the most recent release in order to have the latest and most up-to-date gene annotations.
+However, if you are planning to combine your data with a dataset that was processed in the past you may want to use the annotation version that was used previously for greater consistency.
+Once you have decided on a release to use, you can then select an annotation file.
+This should be the most comprehensive annotation that matches the reference genome you are using.
+So if you are using the human primary assembly you would want the comprehensive annotation for the primary assembly (the `gencode.{release}.primary_assembly.annotation.gtf.gz` file from GENCODE or the `Homo_sapiens.GRCh38.{release}.gtf.gz` file from ENSEMBL).
+For something like fly, you would want the annotation matching the toplevel assembly (e.g. `Drosophila_melanogaster.BDGP6.46.{release}.gtf.gz` from ENSEMBL).
+As well as the comprehensive annotations for the primary and toplevel assemblies, and just the reference chromomes, GENCODE also provides "basic" annotations which only include representative transcripts, but we do not recommend using these.
+
+Gene annotations typically provide a primary identifier for each feature as well as a more common name.
+For example, the ENSEMBL ID `ENSG00000254647` corresponds to the `INS` gene which encodes the insulin protein.
+While the gene names may be more familiar and easier to understand it is important to retain and use the primary identifiers as the are unique for a given annotation and are much easier to map between annotation versions or sources.
+
+To take advantage of all the quality control modules implemented in the pipeline, the gene annotation should include a `gene_biotype` field which describes the function of each feature (protein coding, long non-coding etc.).
+This is usually the case for annotations from GENCODE or ENSEMBL but may not be if your annotation comes from another source.
+If your annotation does not include this field, please set the `--skip_biotype_qc` option to avoid running the steps that rely on it.
```

Comment:
```suggestion
To take advantage of all the quality control modules implemented in the pipeline, the gene annotation should include a `gene_biotype` field which describes the function of each feature (protein coding, long non-coding etc.). This is usually the case for annotations from GENCODE or ENSEMBL but may not be if your annotation comes from another source. If your annotation does not include this field, please set the `--skip_biotype_qc` option to avoid running the steps that rely on it.
```